OrbitGl/CaptureWindow.cpp

```diff
@@ -376,19 +376,23 @@ void CaptureWindow::MouseWheelMoved(int a_X, int a_Y, int a_Delta,
   if (delta > m_MaxWheelDelta) m_MaxWheelDelta = delta;
 
   float mousex = a_X;
+  float mousey = a_Y;
 
   float worldx;
   float worldy;
 
   ScreenToWorld(a_X, a_Y, worldx, worldy);
-  m_MouseRatio = static_cast<double>(mousex) / getWidth();
 
   bool zoomWidth = !a_Ctrl;
   if (zoomWidth) {
+    m_MouseRatio = static_cast<double>(mousex) / getWidth();
     time_graph_.ZoomTime(delta, m_MouseRatio);
     m_WheelMomentum = delta * m_WheelMomentum < 0 ? 0 : m_WheelMomentum + delta;
   } else {
-    // TODO: reimplement vertical zoom by scaling track heights.
+    m_MouseRatio = static_cast<double>(mousey) / getHeight();
```

Comment:
m_MouseRatio is being reused for different purposes here, we should probably have separate variables, mouse_x_ratio_ and mouse_y_ratio_ maybe. That being said, I don't think we need it for the vertical case.

Reply:
Maybe we do need it to focus the zoom on the current mouse position, I'd still rather have two distinct variables.

Reply:
Yes, needed. But changing its name!